psi4/src/psi4/fisapt/fisapt.cc

```diff
@@ -3138,6 +3293,42 @@ void FISAPT::felst() {
         }
     }
 
+    // Extern A-atom B interaction
```

Comment:
Clean up the code duplication.

Reply:
I think the code as it is is cleaner. Various indices and variables are used here. Also, the separation between these contributions is more in line with the rest of the code in the function.